app/src/main/java/org/mozilla/focus/ext/Context.kt

```diff
@@ -43,3 +46,14 @@ val Context.installedDate: String
         val installTime = this.packageManager.getPackageInfo(this.packageName, 0).firstInstallTime
         return DateFormat.getDateInstance().format(installTime)
     }
+
+/**
+ * Checks if the current device is a tablet.
+ */
+fun Context.isTablet(): Boolean = resources.getBoolean(R.bool.is_tablet)
+
+/**
+ * Casts [Context] to [Activity].
+ */
+fun Context.asActivity() =
```

Comment:
Could we use this opportunity to make this name more accurate? Something like, `Context.tryAsActivity`, `Context.withMaybeActivity`, etc?